spacy_example_pipeline.py

```diff
@@ -0,0 +1,29 @@
+import spacy
+
+import quickumls
+from quickumls.spacy_component import SpacyQuickUMLS
+
+# setup a spacy pipeline which includes QuickUMLS as a component and nothing else
+nlp = spacy.load('en_core_web_sm')
+
+nlp.remove_pipe('tagger')
+nlp.remove_pipe('parser')
+
+# TODO -- change this from a hard coded path to a small chunk of UMLS that can be legally distributed
```

Comment:
Unlikely we can distribuite any of UMLS, but maybe some files that looks enough like UMLS?

Reply:
Completely agree.  No reason to keep a hard-coded path.  I considered this to be a "rough draft" PR and this item I had not considered.  I will update this.

Reply:
Also, I did comment on the UMLS issue.  A colleague told me once that they **believed** that there was a  small subset of the UMLS that could be distributed.  I'll follow up on that.   In the meantime, I agree that this is not necessary to bring this PR into the repo.  That can be later work.  

Speaking of later work, I'll follow up with you and anyone else interested in what we are planning with a project called [medspacy](https://github.com/medspacy/medspacy).  We'd like to include QuickUMLS as part of the default pipeline that people can install with a single command.

Reply:
OK, I re-read your comment and understood it.  I removed the previous example .py file and put a small example in the README.  I'm open to any adjustments for the README including order or placement.